facebook/build.gradle

```diff
@@ -0,0 +1,57 @@
+apply plugin: 'com.android.library'
+apply plugin: 'kotlin-android'
+apply plugin: 'maven-publish'
+
+android {
+    compileSdkVersion 28
+    defaultConfig {
+        minSdkVersion 21
+        targetSdkVersion 28
+        versionCode = libProperties['VERSION_CODE']
+        versionName libProperties['VERSION_NAME']
+        testInstrumentationRunner "androidx.test.runner.AndroidJUnitRunner"
+    }
+    buildTypes {
+        release {
+            minifyEnabled false
+            proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
+        }
+    }
+}
+
+dependencies {
+    implementation"org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlin_version"
+
+    implementation "io.reactivex.rxjava2:rxjava:$rxJavaVersion"
+
+    implementation 'com.facebook.android:facebook-login:4.37.0'
+
+    androidTestImplementation "junit:junit:$junitVersion"
+    androidTestImplementation "org.mockito:mockito-core:$mockitoVersion"
+    androidTestImplementation "com.google.dexmaker:dexmaker:$dexmakerVersion"
+    androidTestImplementation "com.google.dexmaker:dexmaker-mockito:$dexmakerVersion"
+    androidTestImplementation "androidx.test:rules:$androidTestVersion"
+}
+
+ext {
+    bintrayName = 'facebook'
```

Comment:
If you want to have different  artifactId than the module name (facebook vs rxfacebooklogin) you need to add to  defaultConfig artifactsBaseName property with the value same as the artifactId. I had to do this today in RxOauth project - https://github.com/AckeeCZ/rxoauth/commit/ad498d7c8d67a05cc10724936e477a9e01a00792